test/gc/Dialect/OneDNNGraph/onednn-graph-ops.mlir

```diff
@@ -0,0 +1,99 @@
+// RUN: gc-opt -split-input-file -canonicalize -verify-diagnostics %s | FileCheck %s
+
+//// Matmul
+
+// CHECK-LABEL: @matmul
+func.func @matmul(%arg0: tensor<128x512xbf16>, %arg1: tensor<512x64xbf16>, 
+                  %arg2: tensor<64xbf16>) -> tensor<128x64xbf16> {
+  // CHECK: onednn_graph.matmul 
+  %0 = onednn_graph.matmul %arg0, %arg1, %arg2 
+       : (tensor<128x512xbf16>, tensor<512x64xbf16>, tensor<64xbf16>) -> tensor<128x64xbf16>
+  return %0 : tensor<128x64xbf16>
+}
+
+//// Unary
+
+// CHECK-LABEL: @relu
+func.func @relu(%arg0: tensor<128x512xbf16>) -> tensor<128x512xbf16> {
+  // CHECK: onednn_graph.relu 
```

Comment:
Do we need these at all? What do they test?

Reply:
Test for basic format, attrs, auto verify and canonicalize, etc. Similar to `llvm-project/mlir/test/Dialect/Linalg/named-ops.mlir`